source/Preferences.cpp

```diff
@@ -57,6 +57,7 @@ void Preferences::Load()
 	// values for settings that are off by default.
 	settings["Automatic aiming"] = true;
 	settings["Render motion blur"] = true;
+	settings["Camera Acceleration"] = true;
```

Comment:
Generally what I do with new features is try to keep the default option as it was before.
Its by no means a hard rule but I'd like some extra opinions on this; should this be on by default? I cant view the video from phone so I cant say

Reply:
done